Readability.js

```diff
@@ -724,8 +734,12 @@ Readability.prototype = {
         // Add the score to the parent. The grandparent gets half.
         parentNode.readability.contentScore += contentScore;
 
-        if (grandParentNode)
+        if (grandParentNode && grandParentNode.readability) {
```

Comment:
That's betond my current understanding, but yes, in some circumstances I didn't take the time to fully investigate, `grandParentNode` can be `undefined`.